src/main/webapp/index.html

```diff
@@ -4,15 +4,16 @@
     <meta charset="UTF-8">
     <title>SAY</title>
     <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script src="script.js"></script>
   </head>
   <body>  
     <div>
-      <form>
+      <form onsubmit="return false">
         <p>Enter a YouTube video link:</p>
-        <textarea name="link-input"></textarea>
+        <input name="link-input" required pattern="https?://www.youtube.com/watch\?v=.+"/>
```

Comment:
When I go to a youtube video and click on the share button, I get something like:
https://youtu.be/sPTn0QEhxds

There could be other formats of shortcuts being used by user as well. You may want to consider those cases as well.

Also question: Why is there a "?" in the link?

Reply:
Spoke offline, explicitly stating format it should be in for now, add URL validation later. ? is for param